rpc/client.go

```diff
@@ -49,11 +48,10 @@ var (
 
 // clientRetryOptions specifies exponential backoff starting
 // at 1s and ending at 30s with indefinite retries.
-var clientRetryOptions = retry.Options{
-	Backoff:     1 * time.Second,  // first backoff at 1s
-	MaxBackoff:  30 * time.Second, // max backoff is 30s
-	Constant:    2,                // doubles
-	MaxAttempts: 0,                // indefinite retries
+var clientRetryOptions = retry.Config{
+	InitialInterval: 15 * time.Second, // first backoff at 1s
```

Comment:
Sorry, scratch that. Instead, I'm wondering why `InitialInterval` is the correct name (it suggests.. an interval, but it's really the plain backoff used before jittering). Looks like it should simply be Backoff (as before) or InitialBackoff.